### PR TITLE
Invite component

### PR DESCRIPTION
**Pull request by ralph-dev**

### CODE REVIEW

File: src/Containers/Invite/Invite.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+import { ArrowRight } from "@styled-icons/bootstrap/ArrowRight";
+import { Heading, Paragraph, SmallText } from '../../index'; 
+
+export interface InviteProps { 
+    title: string;
+    heading: string;
+    description: string;
+    footer: string;
+    buttonText: string; 
+    inviteArgs: any[];
```
Comment (ralph-dev) [inline src/Containers/Invite/Invite.tsx:17]: any[]

Array<{label: string etc}>
{label:string, subLabel: string}[]

File: src/Containers/Invite/Invite.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                                onChange=""                     
+                            />
+                        </IDiv>
+                    </FormGroup>  
+            ),
+        )
+    };
+
+    return (
+        <Wrapper>
```
Comment (ralph-dev) [inline src/Containers/Invite/Invite.tsx:68]: <Wrapper {...props}>

File: src/Containers/Invite/Invite.tsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+            <Row {...args}>
```
Comment (ralph-dev) [inline src/Containers/Invite/Invite.tsx:69]: <Row {...rowProps}

File: src/Containers/Invite/Invite.tsx [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    buttonText: string; 
+    inviteArgs: any[]; 
+}
+export const Invite: React.FC<InviteProps> = ({ 
+    title,
+    heading,
+    description,
+    footer,
+    buttonText,
+    inviteArgs,
```
Comment (ralph-dev) [inline src/Containers/Invite/Invite.tsx:25]: rowProps,

File: src/Containers/Invite/Invite.tsx (hunk 1 above)
Comment (bignatue) [reply src/Containers/Invite/Invite.tsx]: Hi Ralph, line 17 has been modified.
`inviteArgs: {
        label: string;
        placeholder: string;
        type: string;
        subLabel?: string;
    }[]; `

File: src/Containers/Invite/Invite.tsx (hunk 2 above)
Comment (bignatue) [reply src/Containers/Invite/Invite.tsx]: Extra div was also removed.

**Comment by bignatue** [comment]
Output preview
preview

**Review by ralph-dev — COMMENTED** [review]
Code quality needs to be improved will fix on a call

**Review by ralph-dev — COMMENTED** [review]
The invite component still doesn't have an onSumbit prop? How will you send this data to a different server during multiple implements?

**Review by ralph-dev — COMMENTED** [review]
Don't write props.anything use destructing line the other components

**Pull request merged**